python/models.md

```diff
@@ -0,0 +1,9 @@
+# Model style
+
+```
+title = models.CharField(
+    max_length=100,
+    blank=True,
+    null=True,
```

Comment:
I'm really not a fan of the dangling comma, but if we do decide to use it i'd suggest we start using it in JS too for the same reason's it gets used here